src/itest/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretIntegrationTests.java

```diff
@@ -86,11 +87,17 @@ void shouldAddNewComputeSecrets() {
         addNewRequesterSecret(requesterAddress, requesterSecretKey, secretValue);
 
         // Check the new secrets exists for the API
-        ResponseEntity<ApiResponseBody<String>> appDeveloperSecretExistence = apiClient.isAppDeveloperAppComputeSecretPresent(appAddress, appDeveloperSecretIndex);
-        Assertions.assertThat(appDeveloperSecretExistence.getStatusCode()).isEqualTo(HttpStatus.NO_CONTENT);
+        try {
+            apiClient.isAppDeveloperAppComputeSecretPresent(appAddress, appDeveloperSecretIndex);
+        } catch (FeignException e) {
```

Comment:
I'm not sure why we'd encounter an exception if we actually have a `NO_CONTENT` (204) return code.

Reply:
The convention of the Feign library is to throw a `FeignException` as soon as HTTP status is not 200.

Reply:
That's strange but OK :)